app/src/main/java/nerd/tuxmobil/fahrplan/congress/repositories/SessionsTransformer.kt

```diff
@@ -58,6 +58,7 @@ class SessionsTransformer @VisibleForTesting constructor(
             } else {
                 RoomData(
                     roomName = roomName,
+                    roomGuid = sessions.first().roomIdentifier,
```

Comment:
Are you sure the `first` session has the correct `roomIdentifier` for each session? Would be good to have a unit test which covers this.